playbooks/library/raxmon.py

```diff
@@ -139,15 +150,32 @@ def create_agent_token(module, conn, entity):
         module.fail_json(msg=msg)
 
 
+def delete_entity(module, conn, entity):
+    entities = _get_entities(conn, entity)
+    for entity in entities:
+        try:
+            conn.delete_entity(entity)
```

Comment:
Is the goal here to delete all entities or just the given entity. I thought it's the latter?